### DISCUSSION TITLE
Fitting Hospitalizations, ICU, and Deaths to Measured Data

### DISCUSSION THREAD

**Discussion by lewleib** [neherlab/covid19_scenarios#665] [General]
Thank you for an easy to use and useful tool.  I am trying to use your model to predict future hospital uses in California.
By adjusting R0 and intervention schedules I can get an approximate fit to measured data for ICU use and cumulative deaths.  However, the measured hospitalization numbers are much higher than the calculated severely ill predictions.  I've tried to adjust hospital stay days but get big swings in ICU overload.
Can the model calculate hospitalizations?
What parameters impact the relationship between hospitalization and ICU use?  Thanks so much.

**Comment by nnoll** [comment]
There are a few key parameters that affect the accumulative of hospitalized & critical patients. You have identified some: namely the length of the hospital stay and the length of the ICU stay. Increasing the length of hospital stay will _slow_ the usage of the ICU wards as patients reside in general care _longer_ before transitioning to the ICU. 

The other way to change the overall flux of patients from the hospital to critical care is to adjust the assumptions of our severity table - specifically the critical column. This directly controls the fraction of hospitalized patients in each demographic expected to require critical care. In short, this column will control the _relative amount_. This will be most sensitive to the older demographics.

If you are willing (and its not private), I can provide more targeted help if I can see the data you are trying to fit to.

**Comment by lewleib** [comment]
Hi Nicholas,

Thanks for the quick response.

I started trying to fit your predictive model to the measured data shown on your scenario graph for the State of California(see below).  Now, I am also looking at the more up to date information available from the State of California: https://public.tableau.com/views/COVID-19PublicDashboard/Covid-19Public?%3Aembed=y&%3Adisplay_count=no&%3AshowVizHome=no.  This includes cumulative and daily cases, deaths, hospitalization, and ICU utilization(also available in tabulated form).

Any help you can provide would be appreciated.  I will dive in to manipulating the severity table.

Kind regards,

Lew
